packages/engine-lightning/src/sdks/sdk-lightning/index.ts

```diff
@@ -138,8 +139,8 @@ const _configureProject = (c) =>
         addSystemInjects(c, injects);
 
         const configFile = platform === TIZEN ? 'config.xml' : 'appinfo.json';
-        const file = path.join(getPlatformProjectDir(c), configFile);
-        writeCleanFile(file, file, injects, null, c);
+        const file = path.join(getPlatformProjectDir(c)!, configFile);
```

Comment:
I see mixed `getPlatformProjectDir(c)!` and `getPlatformProjectDir(c) || ''`, maybe stick with one?

Reply:
done